cmd/kubeadm/app/util/env.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"os"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// GetProxyEnvVars builds a list of environment variables to use in the control plane containers in order to use the right proxy
```

Comment:
i think this comment now needs a reword - it returns a generic v1.EnvVar that anything can consume (not only CP pods).